index.html

```diff
@@ -3294,6 +3549,16 @@ <h2>
         using one of the following options:
       </p>
       <dl>
+        <dt>
+          Localized values in the manifest:
+        </dt>
+        <dd>
+          Authors can provide [=localized values=] for the [=localizable
+          members=] of the [=manifest=]. The user agent should pass all
```

Comment:
should or SHOULD?

Reply:
This section is informative, so I assume we should avoid RFC keywords here.